compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/controlcode_to_transaction.mlir

```diff
@@ -15,7 +15,7 @@ module {
 // -----
 
 // CHECK:       0x06030100
-// CHECK:       0x00000105
+// CHECK:       0x00000104
```

Comment:
It was previously hardcoded as `0x00000105`, which was wrong as the last byte stands for the number of columns, and we are using `npu1_4col`.